example/README.md

```diff
@@ -0,0 +1,2 @@
+This example implements a gRPC Go Echo client and server which use self - signed X.509 certificates and s2av2TransportCreds(implementation of
```

Comment:
2 micro-nits:

1. Please remove spaces between self and signed, i.e. it should all be one word "self-signed".
2. Please leave a space between s2av2TransportCreds and the start of the parenthesis.